README.md

```diff
@@ -1 +1,264 @@
-libsyscall_intercept - coming soon!
+# syscall_intercept
+
+Userspace syscall intercepting library.
+This repository is soon going to be deleted
+
+# How to build #
+
+Building libsyscall_intercept requires cmake.
+Example:
+```sh
+cmake path_to_syscall_intercept -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=clang
```

Comment:
It is an example, and ```path_to_syscall_intercept``` is supposed to mean the path to the sources. What would be a better example?